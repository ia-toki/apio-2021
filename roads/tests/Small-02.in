c10234cabdfd6eae47773fb6f163e0350fc797e1
200
126 36 854580157
194 138 174351178
144 118 18591252
94 178 131895119
103 61 545239983
150 171 722872344
130 171 234326384
78 130 88895337
118 190 347800128
102 92 313023597
161 164 755671959
128 13 683463430
163 134 203051546
195 104 745004613
37 93 318552670
47 188 141353461
114 0 896418762
89 162 811330898
105 141 449784731
80 119 753057495
184 38 485457922
192 98 558141237
51 167 966529896
125 30 456662879
158 70 162388433
138 164 686265282
82 97 913774686
193 197 74171027
63 62 102992620
149 169 124400960
123 102 148149211
59 153 785172232
87 116 91855849
3 148 329056934
134 88 772629564
175 69 443978025
86 154 257201278
33 180 694631391
104 193 780578594
187 14 839370095
38 82 334030790
159 167 40085510
111 128 710354353
25 179 142863018
35 163 872178299
18 122 789820842
155 143 699016908
132 27 228210230
99 40 257162556
88 96 375162123
100 32 326441306
13 70 617856582
22 42 447741128
156 141 564880474
27 184 30478468
65 47 879873053
24 32 402992705
196 57 387032061
6 144 157702643
67 33 994464335
122 141 465407338
84 34 418547400
28 41 201587734
91 155 467161860
174 153 625920745
32 65 200891972
106 0 889634394
178 82 23143989
77 161 750104197
97 191 787043051
40 127 344886639
198 44 917415405
131 166 241305448
139 107 598526569
72 69 717051785
120 28 419234952
135 66 491391069
56 19 235835868
68 26 664447644
96 5 940439798
166 35 387029010
21 142 822339605
107 65 305619615
75 103 552960249
57 141 246872407
12 116 188237269
152 33 571923402
172 125 144533346
191 179 675295069
177 81 126735888
154 133 2548439
109 137 359768663
157 151 960105464
143 32 113911340
76 152 801418051
164 97 371675536
117 143 458203613
66 22 156260418
0 4 444760974
5 157 362179733
140 24 178189028
10 0 616882524
146 36 124367222
162 78 358481665
29 33 100330566
49 88 756212149
171 16 130154048
92 46 609933505
46 40 599799007
11 75 370943740
90 104 141083375
36 124 970975317
39 142 679563629
8 124 777573520
48 177 791535242
43 82 551066382
124 179 851488462
151 150 967583172
14 151 998291546
79 1 454597454
119 183 118737478
60 161 63421980
136 141 487501864
71 17 216138442
160 151 141470134
19 136 218450318
31 120 298630787
141 190 440349387
182 123 331582949
85 19 660480706
165 128 870454008
121 109 881809512
23 199 7566011
167 171 79975250
69 26 40082007
9 40 11615948
137 123 859766144
147 111 480296970
188 77 612219637
64 166 330327926
30 187 149652769
186 192 741475624
148 110 621168356
170 111 242886772
189 11 939199595
168 69 974685340
93 185 667336763
55 144 17173670
173 12 740696142
34 42 485861961
129 154 332978004
116 174 855898625
44 4 814131226
112 66 977554498
145 122 456563976
81 194 875128777
58 35 865107396
115 121 553693453
52 43 356465909
110 40 387780308
183 21 445346850
54 42 773478371
7 166 558362040
83 147 701185998
74 76 891660074
180 146 127468781
179 127 10491699
142 76 144248175
41 109 853308983
70 164 246732680
153 186 212009159
62 93 965483206
181 64 659963459
73 170 962691714
45 32 80963482
108 72 511616881
185 146 961843246
42 134 719467610
53 192 699380946
61 154 527946722
98 78 900058869
20 8 649355139
26 154 44127192
199 61 108208027
95 97 83092987
15 93 368068768
1 84 925694102
133 24 985159538
101 91 576316739
190 157 105765577
127 145 806836665
176 0 698935757
50 115 30543794
2 145 164341189
17 51 531772142
169 180 165724391
113 88 173084497
197 152 389691982
4 11 362192680
