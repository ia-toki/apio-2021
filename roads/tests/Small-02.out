064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
95931364777 44461319111 16039412826 4614982035 1227110637 246872407 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
